osbenchmark/builder/models/providers.py

```diff
@@ -0,0 +1,6 @@
+from enum import Enum
+
+
+class Provider(str, Enum):
```

Comment:
How do you feel about calling this something like `InfraProvider` or `ClusterInfraProvider` instead?

Reply:
That's more clear, will change